### PR TITLE
Autism pins!

### PR DESCRIPTION
**Pull request by Terraspark4941**
## About the PR

Adds two new identity pins: the autism infinity pin, and its gold variant

## Why / Balance

I saw the LGBTQ pins and when "What if autism, too?"

## Media

image
image
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Autism pins have been added to maintenance loot!

### CODE REVIEW

**Comment by lzk228** [comment]
i think it's against the server rules, because you cant call someone autistic and other slurs

**Comment by nikthechampiongr** [comment]
> i think it's against the server rules, because you cant call someone autistic and other slurs

At least in the official ss14 servers we are not gonna ban you for wearing an autism pin. That rule is for people that use it in a derogatory way.

**Comment by UbaserB** [comment]
> i think it's against the server rules, because you cant call someone autistic and other slurs

how is autistic a slur? sure, neurodivergent may be a better word but autism is a real thing and i find this pr really nice as someone in the spectrum :)

**Comment by potato1234x** [comment]
Why are they so huge on the ground make them same size as the other pins

**Comment by Terraspark4941** [comment]
> Why are they so huge on the ground make them same size as the other pins

I liked the size of both big pins, but if it's consensus sure (also I have autism so maybe detailmaxxing is my fault lol)

**Comment by UbaserB** [comment]
i dont like how the worn pin glows when worn though, its a bit messy.

**Comment by Terraspark4941** [comment]
> i dont like how the worn pin glows when worn though, its a bit messy.

Once I get back home, I'll fix the normal pin glowing
However, I think I'll keep the gold pin's glow and possibly clean it up a bit; headcanoning it as real gold

**Comment by Terraspark4941** [comment]
Images have been updated to reflect changes

**Comment by PolterTzi** [comment]
I would suggest just renaming them to "autism pin", "autism awareness" might not sit right with some people.

**Comment by UbaserB** [comment]
> Do we want ASCII smileys in descriptions? I don't know if there's a policy against it, but I'm against it personally (hence this comment).
> 
> Can I get others' opinions on this?

i would say no

**Comment by ps3moira** [comment]
You can't handle the uber instincts of my uber autism, observe.

I would also like to suggest maybe using the autism creature as a pin? Or could be used as a plushie in a different PR.
image

**Comment by Terraspark4941** [comment]
> I would also like to suggest maybe using the autism creature as a pin? Or could be used as a plushie in a different PR.

I think it would do well as a plushie, but that would be out of scope for now

**Comment by KompaniLinge** [comment]
Very good PR (i am biased) (remove the "awareness" part tho.)

**Comment by Terraspark4941** [comment]
> Very good PR (i am biased) (remove the "awareness" part tho.)

Forgot to specify the above commit fixes that
I should add new images to reflect the change

**Comment by CatMagic1** [comment]
> Do we want ASCII smileys in descriptions? I don't know if there's a policy against it, but I'm against it personally (hence this comment).
> 
> Can I get others' opinions on this?

why not? most descriptions aren't exactly ic anyway

**Comment by PJB3005** [comment]
> Do we want ASCII smileys in descriptions?
I don't know if there's a policy against it, but I'm against it personally (hence this comment).

Pink shonk already has one.

**Comment by potato1234x** [comment]
> > why not? most descriptions aren't exactly ic anyway
> 
> My argument is that it encourages players using smileys in chat, which in turn **_is_** against the rules.
> 
> _Eyes see text smiley = brain thinks text smileys are OK => fingers type text smiley in chat._

you cant :) in chat it turns into @smiles

**Comment by smiles** [comment]
@potato1234x  >.>

**Comment by potato1234x** [comment]
Oh shit didn't mean to ping you mb

File: Resources/Textures/Clothing/Neck/Misc/pins.rsi/meta.json [hunk 1]
```diff
@@ -1,74 +1,89 @@
 {
   "version": 1,
-  "license": "CC-BY-SA-3.0",
-  "copyright": "PixelTK leaves his mark on upstream",
+  "license": "CC-BY-SA-3.0, CC-BY-NC-4.0",
```
Comment (UbaserB) [inline Resources/Textures/Clothing/Neck/Misc/pins.rsi/meta.json:3]: I'm pretty sure you can only have one copyright, the test is failing because of it.

Comment (Terraspark4941) [reply Resources/Textures/Clothing/Neck/Misc/pins.rsi/meta.json:3]: How can I keep my own copyright in the same meta.json? Or do I have to make another RSI for it?

Comment (Terraspark4941) [reply Resources/Textures/Clothing/Neck/Misc/pins.rsi/meta.json]: Fixed, each pin has its own RSI now

**Comment by Terraspark4941** [comment]
*coughs
Merge request? Been a while since last activity

**Comment by beck-thompson** [comment]
Oh also make sure to change the change log. It isn't supposed to be IC : [Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html)
(If you look at the recent PRs they are trying to change CL that was done incorrectly 😆)

**Comment by Terraspark4941** [comment]
fixed, i should remember to avoid IC in the future
at the very least is the "Yippee!!" at the end valid? in reference to the creature

**Comment by MACMAN2003** [comment]
imo the autism creature is overrated, i think it's probably best to leave it out

**Comment by KompaniLinge** [comment]
> imo the autism creature is overrated, i think it's probably best to leave it out

As an autistic person, I agree. I find it to be overused and boring, but ultimately its not a dealbreaker. I'd prefer for it to be changed to something from ingame tho (weh perchance?)

**Comment by Djungelskog2** [comment]
I love the TBH creature but i dont mind if it doesnt get included

**Comment by Terraspark4941** [comment]
> (weh perchance?)

definitely not, weh is a lizor/species inclusive symbol, and i don't think it fits in with autism (even if lizards are such silly goobers)
i think i will remove the TBH reference, and mark as ready to merge

**Pull request merged**